src/toolbox.ts

```diff
@@ -30,77 +31,84 @@ enum ValidResizeValues {
     SMALL = "s",
     LARGE = "l",
     INCREMENT = "inc",
-    DECREMENT = "dec"
+    DECREMENT = "dec",
 }
 
-const toolboxDividerDiv = "<div class=toolbox-divider></div>"
+const toolboxDividerDiv = "<div class=toolbox-divider></div>";
 
 /** Chains the replaceAll method and the toLowerCase method.
  *  Optionally concatenates a string at the end of the method.
-  */
-String.prototype.replaceLowerConcat = function(before: string, after: string, concat_string: string = null) {
-    
-    if (typeof(concat_string) === "string") {
-        return this.replaceAll(before, after).toLowerCase().concat(concat_string)
+ */
+String.prototype.replaceLowerConcat = function (
+    before: string,
+    after: string,
+    concat_string: string = null,
+) {
+    if (typeof concat_string === "string") {
+        return this.replaceAll(before, after)
+            .toLowerCase()
+            .concat(concat_string);
```

Comment:
Alright this one is definitely going to be controversial. I'm not a big fan of chained function calls to begin with, but this does end up being way more readable than storing each intermediate step (although notably not super debuggable). 

When I encounter this (usually due to a certain library preferring this way of doing things), I tend to prefer fully unwrapped formatting like this:
```typescript
return this.replaceAll(
    before, 
    after,
).toLowerCase(
).concat(
    concat_string,
);
```
(the `toLowerCase` doesn't have to be unwrapped, just demonstrating). 

This is left-aligned with the existing indentation, and provides visual grouping for each step of the process. `prettier` won't let this slide, but ESLint and the "Typescript Language Features" formatter in `vscode` don't mind this layout.

Just a bare tab with each function call on it's own line looks a bit ugly to me. I certainly won't die on this hill - if this is the way we want to do it I'll figure out how to get over myself. Thoughts?